src/main/resources/version.properties

```diff
@@ -0,0 +1,5 @@
+version=${project.version}
```

Comment:
I understand tha these `${...}`'s link this with the content of the pom.xml file. Is that correct?